volttron/platform/vip/rmq_connection.py

```diff
@@ -309,32 +308,71 @@ def send_vip_object(self, message, flags=0, copy=True, track=False):
             platform = self._instance_name
 
         destination_routing_key = "{0}.{1}".format(platform, message.peer)
+        user = getattr(message, 'user', self._rmq_userid)
+        msg_id = getattr(message, 'id', b'')
+        self._send_via_rmq(destination_routing_key, message.subsystem, message.args, msg_id, user)
+
+    def send_vip(self, peer, subsystem, args = None, msg_id = b'',
+                 user = b'', via = None, flags = 0, copy = True, track = False, platform=None):
```

Comment:
please modify so that there are not spaces between the argument and the value....so args=None NOT args = None.

Reply:
Done